src/index.ts

```diff
@@ -31,13 +31,16 @@ const startsWith = (str: string, needle: string) => {
   return str.substring(0, needle.length) === needle;
 };
 
+const defaultIsSearchPath = (path: string) => startsWith(path, "/recherche") || startsWith(path, "/search")
+
 // initialize the tracker
 export function init({
   url,
   siteId,
   jsTrackerFile = "matomo.js",
   phpTrackerFile = "matomo.php",
   excludeUrlsPatterns = [],
+  isSearchPath = defaultIsSearchPath
 }: InitSettings): void {
```

Comment:
`InitSettings` also needs to be changed, to know about this. Should I send a new PR?

Reply:
Yes, backward compatible if possible

Reply:
I added the type in `InitSettings`.

This change is backward compatible as the value might be undefined and has a default value.

It is "forward" compatible if we want to add the search parameters later too.